aptos-move/framework/aptos-framework/sources/fungible_asset.move

```diff
@@ -776,6 +800,15 @@ module aptos_framework::fungible_asset {
             event::destroy_handle(withdraw_events);
             event::destroy_handle(frozen_events);
         };
+
+        if (features::is_fa_mint_burn_events_enabled()) {
```

Comment:
for all the places, let's also check it is not APT

maybe create a local functions here should_emit_mint_burn_events(asset_addr)

to check both feature flag and whether it is not APT